### PR TITLE
Deploy now can throw git message from stderr

### PR DESCRIPTION
**Pull request by ytyubox**
Maybe because I was using multiple computers over Publish, I came across lots of deploying problems, sometimes I needed to `cd .publish/GitDeploy/.git` to know what's happening, this PR fix the error feeding.

### CODE REVIEW

**Comment by ytyubox** [comment]
Please review, no idea if the format is ok.

**Comment by JohnSundell** [comment]
Just use the same indentation as in the rest of the code base. After that's done, I'll review.

**Comment by ytyubox** [comment]
done

File: Tests/PublishTests/Tests/DeploymentTests.swift [hunk 1]
```diff
@@ -78,14 +78,42 @@ final class DeploymentTests: PublishTestCase {
         let indexFile = try remote.file(named: "index.html")
         XCTAssertFalse(try indexFile.readAsString().isEmpty)
     }
+	func testGitDeploymentMethodWithError() throws {
```
Comment (sindresorhus) [inline Tests/PublishTests/Tests/DeploymentTests.swift]:
```suggestion

	func testGitDeploymentMethodWithError() throws {
```

File: Tests/PublishTests/Tests/DeploymentTests.swift [hunk 2]
```diff
@@ -78,14 +78,42 @@ final class DeploymentTests: PublishTestCase {
...
+        CommandLine.arguments.append("--deploy")
+
+        XCTAssertThrowsError(
+            try publishWebsite(in: repo,
+                               using: [
+                                .deploy(using: .git(remote.path))
+            ]),
+            "Expected an error to be thrown") {
+                (error) in
+                XCTAssert(error is PublishingError)
```
Comment (sindresorhus) [inline Tests/PublishTests/Tests/DeploymentTests.swift]: There's some funky indentation and formatting here.

File: Tests/PublishTests/Tests/DeploymentTests.swift (hunk 1 above)
Comment (ytyubox) [reply Tests/PublishTests/Tests/DeploymentTests.swift]: @sindresorhus thank you so much!

**Comment by JohnSundell** [comment]
Post-merge fixes: https://github.com/JohnSundell/Publish/pull/35

**Comment by rollr76518** [comment]
great.

**Pull request merged**